README.rst

```diff
@@ -1,3 +1,19 @@
+   **Warning**
+
+   OpenCensus and OpenTracing have merged to form
+   `OpenTelemetry <https://opentelemetry.io>`__, which serves as the
+   next major version of OpenCensus and OpenTracing.
+
+   OpenTelemetry has now reached feature parity with OpenCensus, with
+   tracing and metrics SDKs available in .NET, Golang, Java, NodeJS, and
+   Python. **All OpenCensus Github repositories will be archived**. We
+   encourage users to migrate to OpenTelemetry.
+
+   To help you gradually migrate your instrumentation to OpenTelemetry,
+   bridges are available in Java, Go, Python, and JS. `Read the full
+   blog post to learn
```

Comment:
@lzchen @aabmass I believe the Python bridge is missing coverage of all the signals. Do we think we should mention it here or just point to the blog post that provides more details?

Reply:
tracing is implemented but metrics is not. I do not believe we need a logging bridge since the api is essentially a wrapper around the python logging library.

Reply:
Yes, ideally someone can work on the MetricProducer and bridge before we fully archive this (although IIRC it is a reversible operation)